### DISCUSSION TITLE
What is the "x86" folder in the 64-bit version?

### DISCUSSION THREAD

**Discussion by ghost** [winsiderss/systeminformer#1531] [Questions / Answers] [answered]
Good evening, Developers  👋🏼
Is the "x86" folder needed in the 64-bit version of System Informer? If so, what is it responsible for?

![x86 folder](https://user-images.githubusercontent.com/61148267/206866797-551c3b13-8eda-4099-a9e6-3e6f3f3bba8a.png)

**Comment by dmex** [comment, marked as answer]
Hello

> Is the "x86" folder needed in the 64-bit version

Yes.

> If so, what is it responsible for?

Those two binaries are responsible for querying information from any 32bit processes running on 64bit Windows viewing .NET Framework statistics, Process Heaps, GDI objects or other information contained within 32bit processes. 

There's a large amount of data the OS and Applications can't communicate across boundaries because vendors never supported 64bit when these components were designed or because of differences in architecture and backwards compatibility.  So when you're viewing or using features on any 32bit processes you'll see the x86 binary created as a service and used to copy information into the 64bit process.
